### DISCUSSION TITLE
Massive Open Online Course

### DISCUSSION THREAD

**Discussion by Loksly** [d3/d3#3482] [Q&A]
Hi!

I'm looking forward to learn Data Visualization using d3 in _Observable_ platform. I've seen there are tutorials, youtube sessions, documentation and so on, but I haven't seen a massive course in a platform like edx or coursera. Coursera has some d3 courses but these are nor official nor related to Observable platform.

Are there any plans for this in a near future?

Thank you for your time.

**Comment by curran** [comment]
I'm doing a free course at the moment https://vizhub.com/forum/t/get-it-right-in-black-white-index/110
